experimenter/experimenter/targeting/constants.py

```diff
@@ -2244,18 +2244,39 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
-AD_BLOCKERS_INSTALLED = NimbusTargetingConfig(
-    name="Ad blockers installed",
-    slug="ad_blockers_installed",
-    description="Users who have installed an adblocker.",
+HAS_BING_AS_CURRENT_DEFAULT_SEARCH_ENGINE_AND_AD_BLOCKERS_INSTALLED = NimbusTargetingConfig(
+    name="Has Bing as current default search engine and ad blockers installed",
+    slug="has_bing_as_current_default_search_engine_and_ad_blockers_installed",
+    description="Users with bing as current default search engine and has an adblocker installed.",
     targeting="""
-        addonsInfo.addons['uBlock0@raymondhill.net'] != null ||
+        searchEngines.current =='bing' &&
+        (addonsInfo.addons['uBlock0@raymondhill.net'] != null ||
         addonsInfo.addons['adblockultimate@adblockultimate.net'] != null ||
         addonsInfo.addons['firefox@ghostery.com'] != null ||
         addonsInfo.addons['jid1-NIfFY2CA8fy1tg@jetpack'] != null ||
         addonsInfo.addons['{d10d0bf8-f5b5-c8b4-a8b2-2b9879e08c5d}'] != null ||
         addonsInfo.addons['jid1-MnnxcxisBPnSXQ@jetpack'] != null ||
-        addonsInfo.addons['{74145f27-f039-47ce-a470-a662b129930a}'] != null
+        addonsInfo.addons['{74145f27-f039-47ce-a470-a662b129930a}'] != null)
```

Comment:
Oh you know, rather than replace the `AD_BLOCKERS_INSTALLED` entirely, it's preferred to leave it intact (to keep existing experiments that refer to it intact), and build a new targeting that refers back to it like

```py
HAS_BING_AS_CURRENT_DEFAULT_SEARCH_ENGINE_AND_AD_BLOCKERS_INSTALLED = ...
    targeting=(
        f"{AD_BLOCKERS_INSTALLED.targeting} &&"
        ...
    ),
    ...
```